spec/fixtures/files/csv/valid.csv

```diff
@@ -8,3 +8,4 @@ Tranquility
 Enigma
 Melody
 Whimsical, oh
+VPN
```

Comment:
Curious why do we need to add this Keyword. Would the spec have less change if this is not added?

Reply:
Because the `valid_bing.html` is an example with the keyword VPN, so I added it. Other keywords won't produce any ads result.

Reply:
In the network stub, if we return `valid_bing.html` always then there will be no difference which word we put here 🤔 . I think stubbing for one html result it is already enough as the code would not know if the html return is related to the keyword or not.

Reply:
Yep, however I still want to it consistent 😁